scripts/windows-native-ci.ps1

```
@@ -430,6 +430,135 @@ function Limit-WindowsNativeText([AllowNull()][string]$Text, [int]$MaxBytes = 10
+        [StringComparer]::Ordinal
+    )
+    $failedPackages = [Collections.Generic.List[object]]::new()
+    $failedPackageNames = [Collections.Generic.HashSet[string]]::new(
+        [StringComparer]::Ordinal
+    )
+    $failedTestsOmitted = $false
+    $reader = [IO.StringReader]::new($Text)
+    try {
+        while ($null -ne ($line = $reader.ReadLine())) {
+            try {
+                $event = $line.TrimStart([char]0xFEFF) | ConvertFrom-Json -ErrorAction Stop
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'event' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)